src/main/webapp/js/about-us-map.js

```diff
@@ -8,20 +8,39 @@ function createDefaultMap(){
 
 function createMapWithDefaultMarkers(){
   createDefaultMap();
-  createMarker(1.280547, 103.844502, 'Maxwell Food Centre', '1 Kadayanallur St 069184');
-  createMarker(1.308398, 103.885808, 'Old Airport Road Food Centre', '51 Old Airport Rd, Singapore 390051');
-  createMarker(1.331814, 103.938867, 'Bedok 85 Market', '85 Bedok North Street 4, Singapore 460085');
+  createMarker(1.280547, 103.844502, 'Maxwell Food Centre', '1 Kadayanallur St 069184',
+                'http://www.okoguide.com/upload/img/objects/Singapore/SINGAPORE_8282.jpg');
+  createMarker(1.308398, 103.885808, 'Old Airport Road Food Centre', '51 Old Airport Rd, Singapore 390051',
+                'https://sethlui.com/wp-content/uploads/2018/07/old-airport-road-food-centre-11-800x532.jpg');
+  createMarker(1.331814, 103.938867, 'Bedok 85 Market', '85 Bedok North Street 4, Singapore 460085',
+                'http://danielfooddiary.com/wp-content/uploads/2019/03/fengshan29.jpg');
+}
+
+function infoContent(title, address, url){
+  var infoDiv = document.createElement('div');
+  infoDiv.classList.add('info');
+  var name = document.createElement('h1');
+  name.innerHTML = title;
+  var img = document.createElement('img');
+  img.src = url;
+  var a = document.createElement('p');
+  a.innerHTML = 'Address: ' + address;
+  infoDiv.appendChild(img);
+  infoDiv.appendChild(name);
+  infoDiv.appendChild(a);
+  return infoDiv;
 }
 
 /* Add landmark to map */
-function createMarker(lat, lng, title, address) {
+function createMarker(lat, lng, title, address, url) {
```

Comment:
nit: feel like this should be `image_url` or similar since it needs to be a link to an image